packages/containers/src/DeleteResource/DeleteResource.container.js

```diff
@@ -92,7 +94,7 @@ export class DeleteResource extends React.Component {
 				validateAction={validateAction}
 			>
 				<div>
-					<Trans i18nKey={i18nKey}>
+					<Trans i18nKey={i18nKey} parent="div">
```

Comment:
Even with the doc link I don't understand why this is needed.
If it add a div to wrap the result, the parent `div` can be removed right ?